tests/audio/test_loading.py

```diff
@@ -3,3 +3,7 @@
 def test_missing_file(audiomgr):
     sound = audiomgr.get_sound('/not/a/valid/file.ogg')
     assert str(sound).startswith('NullAudioSound')
+
+def test_found_file(audiomgr):
```

Comment:
Let's rename this to `test_load_ogg_vorbis()` to make adding other file types (e.g., ogg opus) more obvious.

Reply:
This path is relative to the current working directory; we should make it relative to `__file__` using something like:

```python
import panda3d.core
path = p3d.Filename.from_os_specific(os.path.join(os.path.dirname(__file__), '..', '..', 'samples', 'music-box', 'music', 'musicbox.ogg'))
```

Or, maybe a little cleaner:
```python
path = os.path.dirname(__file__) + '../../samples/music-box/music/musicbox.ogg'
```

Reply:
Okay, Got it.